spring-integration-azure/src/main/java/com/microsoft/azure/spring/integration/storagequeue/StorageQueueTemplate.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+
+package com.microsoft.azure.spring.integration.storagequeue;
+
+import com.microsoft.azure.spring.integration.storagequeue.factory.StorageQueueFactory;
+import com.microsoft.azure.storage.StorageException;
+import com.microsoft.azure.storage.queue.CloudQueue;
+import com.microsoft.azure.storage.queue.CloudQueueMessage;
+import org.springframework.util.Assert;
+
+
+public class StorageQueueTemplate implements StorageQueueOperation {
+    private final StorageQueueFactory storageQueueFactory;
+    private final CloudQueue cloudQueue;
+
+    StorageQueueTemplate(StorageQueueFactory storageQueueFactory, String queueName) {
+        Assert.hasText(queueName, "queueName can't be null or empty");
+        this.storageQueueFactory = storageQueueFactory;
+        this.cloudQueue = storageQueueFactory.getQueueCreator().apply(queueName);
+    }
+
+    @Override
+    public boolean add(CloudQueueMessage cloudQueueMessage) {
+        try {
+            cloudQueue.addMessage(cloudQueueMessage);
+            return true;
+        } catch (StorageException e) {
+            e.printStackTrace();
```

Comment:
Please use log.error

Reply:
If you just return false, User have no idea why this add failed and whether he or she should retry.

Reply:
You need to have a clear definition of what this return value means in the interface.